app/collections/show-list.cjsx

```diff
@@ -140,14 +184,35 @@ module.exports = React.createClass
           meta = @state.subjects[0].getMeta()
 
           <div>
-            <div className="collection-subjects-list">
+          {if @state.selecting
```

Comment:
sounds good -> updated so the Select Subjects button won't show unless signed in.